src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -0,0 +1,301 @@
+## 1.15.0a1 (tbd)
```

Comment:
update the changelog - new release is 1.14.0a2

Reply:
Yep, after the code freeze will be copying over the latest code from `src/snowflake/snowpark/modin/*`, `tests/unit/modin/*`, `tests/integ/modin/*` which should cover this.